src/main/java/uk/gov/hmcts/reform/pip/data/management/errorhandling/GlobalExceptionHandler.java

```diff
@@ -67,8 +68,18 @@
         return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(exceptionResponse);
     }
 
-    @ExceptionHandler(EmptyRequestHeaderException.class)
-    public ResponseEntity<ExceptionResponse> handle(EmptyRequestHeaderException ex) {
+    @ExceptionHandler(HeaderValidationException.class)
+    public ResponseEntity<ExceptionResponse> handle(HeaderValidationException ex) {
+
+        ExceptionResponse exceptionResponse = new ExceptionResponse();
+        exceptionResponse.setMessage(ex.getMessage());
+        exceptionResponse.setTimestamp(LocalDateTime.now());
+
+        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(exceptionResponse);
+    }
+
+    @ExceptionHandler(PayloadValidationException.class)
+    public ResponseEntity<ExceptionResponse> handle(PayloadValidationException ex) {
```

Comment:
I still think we could just have the 1 handler since these 2 types of validation exceptions are handled the same?

Reply:
Ah, so sonarqube was raising everything inheriting from ValidationException as a code smell, so I slightly updated the exception tree to seperate out Header + Payload validations. We now have

RuntimeException -> HeaderValidationException -> <HeaderExceptionsGoHere>
RuntimeException -> PayloadValidationException -> <PayloadExceptionGoesHere>